app/models/concerns/control_objective_items/effectiveness.rb

```diff
@@ -14,6 +14,23 @@ def effectiveness
     scores.empty? ? 100 : average.round
   end
 
+  def previous_effectiveness
+    r = Review.list.find(review_id).previous
```

Comment:
Enano, si tiene un review_id tiene un belongs_to, por lo que podés hacer:
```suggestion
    r = review.previous
```

Y no seas tacaño con el nombre de las variables, está bien en un script, pero en el código que va a quedar... a no ser que haya un buen motivo mejor no hacerlo.